libs/s25main/world/GameWorld.cpp

```diff
@@ -199,7 +199,7 @@ void GameWorld::BuildRoad(const unsigned char playerId, const bool boat_road, co
         return;
     }
 
-    // Gucken, ob der Weg überhaupt noch gebaut werden kann
+    // See if the path can still be built at all
```

Comment:
We should stick to "road" instead of sometimes using "street" or "path" instead

```suggestion
    // See if the road can still be built at all
```